src/Examples/Akka.Hosting.Maui/MauiProgram.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace Akka.Hosting.Maui
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+                .UseMauiApp<App>()
+                .ConfigureFonts(fonts =>
+                {
+                    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                    fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+                });
+
+            builder.Services
+                .AddAkka("TestSys", config =>
+                {
+                    config.WithActors((system, registry) =>
+                    {
+                        var echo = system.ActorOf(ClickActor.Props);
+                        registry.Register<ClickActor>(echo);
+                    });
+                })
+                .AddTransient<IMauiInitializeService, AkkaInitializer>()
```

Comment:
Register a `IMauiInitializeService` to start Akka hosted service.
This is the only way to start a service automatically during start-up in MAUI.